### PR TITLE
feat(build): enable auto build for arm64 binaries

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

- auto build and push arm64 and amd64 binaries

### CODE REVIEW

File: build_image.sh [hunk 1]
```diff
@@ -8,7 +8,15 @@ ARCH=$(uname -m)
...
 make clean
 bash autogen.sh
-./configure --enable-replication
+
+if [ "${ARCH}" = "x86_64" ]; then
+	./configure --enable-replication
+elif [ "${ARCH}" = "aarch64" ]; then
+	./configure --enable-replication --build=arm-linux
+else
+	echo "Unusable architecture: ${ARCH}"
```
Comment (kmova) [inline build_image.sh]: n_: unusable -> unsupported

File: src/arch/arm/rte_atomic.h [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 #if defined(__aarch64__)
 #include "arch/arm/rte_atomic_64.h"
-else
+#else
```
Comment (mynktl) [inline src/arch/arm/rte_atomic.h:10]: why are we removing this?

Comment (shubham14bajpai) [reply src/arch/arm/rte_atomic.h:10]: Not removing, just added `#` which was missing before the `else`.

Comment (mittachaitu) [inline src/arch/arm/rte_atomic.h:10]: :+1:

File: build_image.sh (hunk 1 above)
Comment (shubham14bajpai) [reply build_image.sh]: Done

**Review by mynktl — APPROVED** [review]

**Review by mittachaitu — APPROVED** [review]
changes are good

**Review by kmova — APPROVED** [review]

**Pull request merged**